src/Pages/Articles.tsx

```diff
@@ -33,28 +34,31 @@ const Articles = ({ language, t }) => {
         spinnerService.hide("apiSpinner");
         setArticles(data => ({
           ...data,
-          [currentLanguage]: response.data.items
+          [currentLanguage]: response.data.items as ArticleType[]
         }));
       });
   }, [language]);
 
-  const formatDate = value => {
-    return dateFormat(value, 'mmmm d');
+  const makeFormatDate = (value: string): string => {
+    return formatDate(value, {
+      month: 'long',
+      day: 'numeric',
+    })
   };
 
   let counter = 0;
 
-  const articlesComponent = articles[language].reduce((result, article, index) => {
+  const articlesComponent = articles[language].reduce((result: any, article: Article, index: number) => {
```

Comment:
Result is JSX.Element[]

Reply:
[9ede5d4](https://github.com/Kentico/kontent-sample-app-react/pull/206/commits/9ede5d4e8127366824a28d497840cf0cf5e1cc2e)